src/components/ErrorFilter.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from "react";
+import { sites } from "../siteContent";
+
+interface Status {
+  status: string;
+  value: number;
+}
+
+const ErrorFilter = () => {
+  const [allStatus, setAllStatus] = useState();
+
+  useEffect(() => {
+    const getAllSitesStatus = async () => {
+      const urlList = Object.entries(sites).map(([, pageInfo]) =>
```

Comment:
instead of an `Object.entries` where you only use the second value in the object, I think using Object.values instead can tidy it up!

Reply:
oops completely forgot the existence of `Object.values`